commit 2044513ffe4a9c18e6e2a64f048e05d8b62fa927
Merge: bbcf9cd15767 bdcd93ef9afb
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Thu Aug 6 10:15:28 2020 -0700

    Merge tag 'csky-for-linus-5.9-rc1' of https://github.com/c-sky/csky-linux
    
    Pull arch/csky updates from Guo Ren:
     "New features:
       - seccomp-filter
       - err-injection
       - top-down&random mmap-layout
       - irq_work
       - show_ipi
       - context-tracking
    
      Fixes & Optimizations:
       - kprobe_on_ftrace
       - optimize panic print"
    
    * tag 'csky-for-linus-5.9-rc1' of https://github.com/c-sky/csky-linux:
      csky: Add context tracking support
      csky: Add arch_show_interrupts for IPI interrupts
      csky: Add irq_work support
      csky: Fixup warning by EXPORT_SYMBOL(kmap)
      csky: Set CONFIG_NR_CPU 4 as default
      csky: Use top-down mmap layout
      csky: Optimize the trap processing flow
      csky: Add support for function error injection
      csky: Fixup kprobes handler couldn't change pc
      csky: Fixup duplicated restore sp in RESTORE_REGS_FTRACE
      csky: Add cpu feature register hint for smp
      csky: Add SECCOMP_FILTER supported
      csky: remove unusued thread_saved_pc and *_segments functions/macros

diff --cc tools/testing/selftests/seccomp/seccomp_bpf.c
index 06bd5901be74,8d18a0ddafdd..535720b2592a
--- a/tools/testing/selftests/seccomp/seccomp_bpf.c
+++ b/tools/testing/selftests/seccomp/seccomp_bpf.c
@@@ -1781,7 -1704,7 +1791,7 @@@ void change_syscall(struct __test_metad
  
  #if defined(__x86_64__) || defined(__i386__) || defined(__powerpc__) || \
  	defined(__s390__) || defined(__hppa__) || defined(__riscv) || \
- 	defined(__xtensa__)
 -	defined(__csky__)
++	defined(__xtensa__) || defined(__csky__)
  	{
  		regs.SYSCALL_NUM = syscall;
  	}